test/optimizations/bulkcomm/bharshbarg/stridedAssignDR.chpl

```diff
@@ -110,6 +110,35 @@ proc test(param rank : int, Dom) {
   if printCount then writeln("Performed ", count, " assignments");
 }
 
+proc testReindex(param rank : int, Dom) {
+
+  var A : [Dom] int;
+  var B : [Dom] int;
+
+  var slices = buildSlices(rank, Dom);
```

Comment:
Is this testing a rank-change slice? If not, can you add a test of the similar pattern when including rank changing?

Reply:
There's a test called 'rankChange.chpl' in that same directory, is that adequate?

Reply:
yes